nerdlets/root/function-picker.js

```diff
@@ -1,12 +1,16 @@
 import React from "react"
-import {Stack, StackItem, Radio} from 'nr1'
+import Select from 'react-select'
+import {Stack, StackItem } from 'nr1'
 
 const FUNCTIONS = [
   {fn: 'average', label: 'avg'},
   {fn: 'sum', label: 'sum'},
+  {fn: 'latest', label: 'latest'},
+  {fn: 'median', label: 'median'},
+  {fn: 'max', label:'max'}
```

Comment:
If you replace each instance of `fn` with `value` (in addition to the change from my other comment), the dropdown will work just as expected with updating label and all.